packages/react/test/specs/components/SplitButton/SplitButton-test.tsx

```diff
@@ -0,0 +1,104 @@
+  findIntrinsicElement(wrapper, `.${MenuButton.className} .${Button.className}`)
+const getMenuItems = (wrapper: ReactWrapper): CommonWrapper =>
+  findIntrinsicElement(wrapper, `.${Menu.slotClassNames.item}`)
+const getMenu = (wrapper: ReactWrapper): CommonWrapper =>
+  findIntrinsicElement(wrapper, `.${Menu.className}`)
+
+describe('SplitButton', () => {
+  isConformant(SplitButton)
+
+  describe('open', () => {
+    test('is toggled between true and false on toggle button click', () => {
+      const wrapper = mountWithProvider(<SplitButton menu={mockMenu} button={'test'} />)
+      const toggleButton = getToggleButton(wrapper)
+
+      toggleButton.simulate('click')
+      expect(getMenuItems(wrapper).length).toBe(mockMenu.items.length)
+
+      toggleButton.simulate('click')
+      expect(getMenuItems(wrapper).length).toBe(0)
+    })
+
+    test('is false when clicking menu item', () => {
+      const wrapper = mountWithProvider(
+        <SplitButton menu={mockMenu} button={'test'} defaultOpen={true} />,
+      )
+
+      getMenuItems(wrapper)
+        .at(0)
+        .simulate('click')
+      expect(getMenuItems(wrapper).length).toBe(0)
+    })
+
+    test('is open when Alt+ArrowDown is sent to the main button', () => {
+      const wrapper = mountWithProvider(<SplitButton menu={mockMenu} button={'test'} />)
+
+      getMainButton(wrapper).simulate('keydown', { keyCode: keyboardKey.ArrowDown, altKey: true })
+
+      expect(getMenuItems(wrapper).length).toBe(mockMenu.items.length)
+    })
+
+    test('is open when Alt+ArrowUp is sent to the menu', () => {
+      const wrapper = mountWithProvider(
+        <SplitButton menu={mockMenu} button={'test'} defaultOpen={true} />,
+      )
+
+      getMenu(wrapper).simulate('keydown', { keyCode: keyboardKey.ArrowUp, altKey: true })
+
+      expect(getMenuItems(wrapper).length).toBe(0)
+    })
+
+    test('is open when Escape is sent to the menu', () => {
+      const wrapper = mountWithProvider(
+        <SplitButton menu={mockMenu} button={'test'} defaultOpen={true} />,
+      )
+
+      getMenu(wrapper).simulate('keydown', { keyCode: keyboardKey.Escape })
+
+      expect(getMenuItems(wrapper).length).toBe(0)
+    })
+  })
+
+  test('onMenuItemClick', () => {
+    const onMenuItemClick = jest.fn()
+    const wrapper = mountWithProvider(
+      <SplitButton
+        menu={mockMenu}
+        button={'test'}
+        onMenuItemClick={onMenuItemClick}
+        defaultOpen={true}
+      />,
+    )
+
+    getMenuItems(wrapper)
+      .at(0)
+      .simulate('click')
+    expect(onMenuItemClick).toHaveBeenCalledTimes(1)
+  })
+
+  test('onMainButtonClick', () => {
+    const onMainButtonClick = jest.fn()
+    const wrapper = mountWithProvider(
+      <SplitButton menu={mockMenu} button={'test'} onMainButtonClick={onMainButtonClick} />,
+    )
+
+    getMainButton(wrapper).simulate('click')
+    expect(onMainButtonClick).toHaveBeenCalledTimes(1)
```

Comment:
```suggestion
    expect(onMainButtonClick).toHaveBeenCalledTimes(1)
    expect(onMainButtonClick).toHaveBeenCalledWith(expect.any, expect.objectContaining({ /* ... */ }))
```
Can we add an assert to check that second param is present?